src/components/Moviepage/MoviePage.jsx

```diff
@@ -1,5 +1,69 @@
-import React from "react";
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import Spinner from "../Spinner/Spinner";
 
-export default function MoviePage({ movieInfo }) {
-  return <div>{movieInfo.id}</div>;
+const api_key = process.env.REACT_APP_API_KEY;
+
+export default function MoviePage({ isLoading, setIsLoading }) {
+  const [movieInfo, setMovieInfo] = useState([]);
+  const params = useParams();
+
+  useEffect(() => {
+    fetchMovieData(params.id);
+  }, []);
+
+  async function fetchMovieData(id) {
+    setIsLoading(true);
+    try {
+      const response = await axios.get(
+        `https://api.themoviedb.org/3/movie/${id}`,
+        {
+          params: {
+            api_key: api_key,
+          },
+        }
+      );
+      setMovieInfo(response.data);
+      setIsLoading(false);
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  return (
+    <>
+      {isLoading ? (
+        <Spinner />
+      ) : (
+        <section>
+          <p className="pt-10 pb-4 text-5xl text-center">{movieInfo.title}</p>
+          <div className="flex p-10">
+            <img
+              className="rounded ml-4 w-[400px]"
+              src={`http://image.tmdb.org/t/p/w400/${movieInfo.poster_path}`}
```

Comment:
Every img tag must have alt attribute. Please add it.

Reply:
fixed